.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug Report
+about: An issue used to report a bug to help improve this project.
+title: '[BUG] '
+labels: bug
+assignees: ''
+---
+
+### Overview
+A clear and concise description of the bug.
+
+### Reproduction Procedure
+A series of steps to reproduce the bug.
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected Behavior
+A clear and concise description of the expected behavior.
+
+### Screenshots
+If applicable, add screenshots to help explain your problem.
+
+### Desktop Information
+ - OS: [e.g. MacOS]
+ - IDE [e.g. VSCode, PyCharm]
+ - Python Version [e.g. 3.12]
+ - Pip Version [e.g. 22]
+ - CNBC Version [e.g. 1.0.0]
```

Comment:
Change CNBC to yh-finance

Reply:
Yes. I'll apply this in the next round of revisions.